### PR TITLE
Update Create Your First REST API tutorial to reflect UI changes

### PR DESCRIPTION
**Pull request by RukshiW**

### CODE REVIEW

**Comment by RukshiW** [comment]
Addresses https://github.com/wso2-enterprise/choreo/issues/10644

File: en/docs/tutorials/create-your-first-rest-api.md [hunk 1]
```diff
@@ -52,78 +56,81 @@ The `Statistics` REST API opens on a separate page.
 
 ### Step 1.3: Design the REST API
 
-Let's design the REST API as follows:
+To design the REST API involves specifying how the REST API should function by adding and configuring the required connectors and statements. You can do this by editing the low-code diagram of the REST API or by editing its code.
```
Comment (Nashaath) [inline en/docs/tutorials/create-your-first-rest-api.md:59]: "To design the REST API involves specifying..." doesn't seem correct. Might have to rephrase.

File: en/docs/tutorials/create-your-first-rest-api.md [hunk 2]
```diff
@@ -52,78 +56,81 @@ The `Statistics` REST API opens on a separate page.
...
+In this tutorial, let's design the REST API by updating the low-code diagram as follows:
 
-    Then click **Open Link in New Tab** in the message that appears to indicate that the code server is ready.
+1. To open the REST API component in the Web Editor, click **Edit Code**.
     
     !!! info
         Opening the VS Code Editor may take a little while if you are a first-time user.
 
-2. Add a new construct and an API resource as follows:
+2. Remove the resource configured for the REST API by default by clicking its **Delete** icon.
```
Comment (Nashaath) [inline en/docs/tutorials/create-your-first-rest-api.md:68]: Shall we rephrase this too? 
We can perhaps say
"Remove the resource that is configured by default for the REST API by. . . ."
WDYT?

File: en/docs/tutorials/create-your-first-rest-api.md [hunk 3]
```diff
@@ -52,78 +56,81 @@ The `Statistics` REST API opens on a separate page.
...
+    The specified return type allows the REST API to return the output in either JSON or error? format.
 
-    5. Click **Save API**.
+    Click **Save**.
 
-3. To connect to the COVID-19 API and retrieve data, follow the sub-steps below.
+    The low-code diagram of this resource opens .
 
-    1. Click on the API resource you edited, and then click on the first **+** icon after the **Start** statement.
+4. To connect to the COVID-19 API and retrieve data, follow the sub-steps below.
```
Comment (Nashaath) [inline en/docs/tutorials/create-your-first-rest-api.md:96]:
```suggestion
4. To connect to the COVID-19 API and retrieve data, follow the sub-steps below:
```

File: en/docs/tutorials/create-your-first-rest-api.md [hunk 4]
```diff
@@ -52,78 +56,81 @@ The `Statistics` REST API opens on a separate page.
...
 
-    2. Click **Connector** and then select **Covid19 API**.
+    1. Click **Connector** to add a connector. Then search for the COVID-19 connector, and click on it once it appears in the search results.
 
-    3. In the **Covid19 API Connection** window, enter `covid19Client` as the **Connection Name** and click **Continue to Invoke API**.
+    2. In the **Connector** panel, `covid19Client` as the **Connection Name** and click **Continue to Invoke API**.
 
-    4. In the **Operation** drop-down list, select **getStatusByCountry** and enter details as follows in the other fields:
+       3. In the **Covid19 API Connection** window, enter `covid19Client` as the **Connection Name** and click **Continue to Invoke API**.
```
Comment (Nashaath) [inline en/docs/tutorials/create-your-first-rest-api.md:102]: Numbering alignment doesn't seem right.

Also if the name of the window includes Covid19, shall we report an issue to get it corrected to COVID-19.

File: en/docs/tutorials/create-your-first-rest-api.md [hunk 5]
```diff
@@ -52,78 +56,81 @@ The `Statistics` REST API opens on a separate page.
...
 
-4. To retrieve the population data, connect to the World Bank API as follows:
+           3. Click **Save**.
+
+6. To retrieve the population data, connect to the World Bank API as follows:
 
     1. Click the last **+** icon in the low-code diagram.
 
-    2. Click **Connector** and then select **World Bank API**.
+    2. Click **Connector** to add a connector. Then search for the COVID-19 connector, and click on it once it appears in the search results.
```
Comment (Nashaath) [inline en/docs/tutorials/create-your-first-rest-api.md:131]: This should be World Bank connector. Seems like we've mistakenly mentioned COVID-19

File: en/docs/tutorials/create-your-first-rest-api.md [hunk 6]
```diff
@@ -162,7 +169,7 @@ Let's design the REST API as follows:
 
     3. Click **Save**.
 
-6. To build the JSON payload to be sent as the response and then send the response, follow this procedure:
+8. To build the JSON payload to be sent as the response and then send the response, follow this procedure:
 
     1. To build the JSON payload with data of the total cases per million in the population, add a variable.
```
Comment (Nashaath) [inline en/docs/tutorials/create-your-first-rest-api.md:174]: Since we have just one step here, should we rephrase the step above in a way that we don't need to say ""follow this procedure"?
 WDYT?

File: en/docs/tutorials/create-your-first-rest-api.md (hunk 1 above)
Comment (RukshiW) [reply en/docs/tutorials/create-your-first-rest-api.md]: Rephrased

File: en/docs/tutorials/create-your-first-rest-api.md (hunk 2 above)
Comment (RukshiW) [reply en/docs/tutorials/create-your-first-rest-api.md]: Rephrased.

File: en/docs/tutorials/create-your-first-rest-api.md (hunk 4 above)
Comment (RukshiW) [reply en/docs/tutorials/create-your-first-rest-api.md]: Corrected the numbering. Shall we leave the window name in documentation unchanged for now  until  it is corrected in the UI? Will create an issue for that.

File: en/docs/tutorials/create-your-first-rest-api.md (hunk 6 above)
Comment (RukshiW) [reply en/docs/tutorials/create-your-first-rest-api.md]: Re-arranged the bullet points here.

File: en/docs/tutorials/create-your-first-rest-api.md [hunk 7]
```diff
@@ -29,6 +29,10 @@ In this section, let's develop the application that retrieves COVID-19 related s
 
 4. Click **Create**.
 
+    This take you to the **Components** page shown below.
```
Comment (Nashaath) [inline en/docs/tutorials/create-your-first-rest-api.md:32]: Should be "This takes . . ."

File: en/docs/tutorials/create-your-first-rest-api.md [hunk 8]
```diff
@@ -274,15 +279,15 @@ To deploy the API, follow the steps below:
...
+Once the API is deployed, the **Development** card indicates that the API is active as shown below.
 
 ![Deployed API](../assets/img/tutorials/rest-api/deployed-api.png){.cInlineImage-full}
 
 Now you can test your deployed REST API to check again whether it is working as expected.
 
 ## Step 3: Test
 
-Once you have deployed the Choreo application you can test it via the Open API Console, A cURL command, or Postman.
+Once you have deployed the Choreo application you can test it via the Open API Console, a cURL command, or Postman.
```
Comment (Nashaath) [inline en/docs/tutorials/create-your-first-rest-api.md:290]: The correct term is "OpenAPI Console". Might have to fix this in all applicable places in the doc.

File: en/docs/tutorials/create-your-first-rest-api.md [hunk 9]
```diff
@@ -226,7 +231,7 @@ Let's run the REST API you designed in the VS Code Editor to check whether it ca
     ```
     Now you can try out the REST API.
 
-2. Click **Try it** As a result, a test view opens to the right of the page.
+2. Click **Try it** As a result, a test view opens on the right of the page.
```
Comment (Nashaath) [inline en/docs/tutorials/create-your-first-rest-api.md:234]: Need a full stop after **Try it**.

**Comment by Nashaath** [comment]
There is also a typo on the term Choreo and an occurrence of the term VS Code Editor that needs to be updated in the intro section first bullet point:
"Design a REST API that addresses the described requirement, test it in the VS Code Editor, and then commit it so that it is available in the Chore Console."

File: en/docs/tutorials/create-your-first-rest-api.md [hunk 10]
```diff
@@ -52,78 +56,81 @@ The `Statistics` REST API opens on a separate page.
...
+In this tutorial, let's design the REST API by updating the low-code diagram as follows:
 
-    Then click **Open Link in New Tab** in the message that appears to indicate that the code server is ready.
+1. To open the REST API component in the Web Editor, click **Edit Code**.
     
     !!! info
-        Opening the VS Code Editor may take a little while if you are a first-time user.
+        Opening the Web Editor may take a little while if you are a first-time user.
+
+2. Remove the resource that is configured by default for the REST API by by clicking its **Delete** icon.
```
Comment (Nashaath) [inline en/docs/tutorials/create-your-first-rest-api.md:68]: Need to remove the repetitive by.

File: en/docs/tutorials/create-your-first-rest-api.md [hunk 11]
```diff
@@ -52,78 +56,81 @@ The `Statistics` REST API opens on a separate page.
...
+    The specified return type allows the REST API to return the output in either JSON or error? format.
 
-    4. Click **Advanced** and in the **Return Type** field, enter `json|error` to get the return in `json` or `error1` format.
+    Click **Save**.
 
-    5. Click **Save API**.
+    The low-code diagram of this resource opens .
 
-3. To connect to the COVID-19 API and retrieve data, follow the sub-steps below.
+4. To connect to the COVID-19 API and retrieve data, follow the sub-steps below.
```
Comment (Nashaath) [inline en/docs/tutorials/create-your-first-rest-api.md:96]: We need a colon after below right? Doesn't seem like the suggestion has been applied.

File: en/docs/tutorials/create-your-first-rest-api.md (hunk 7 above)
Comment (RukshiW) [reply en/docs/tutorials/create-your-first-rest-api.md]: Done.

File: en/docs/tutorials/create-your-first-rest-api.md (hunk 10 above)
Comment (RukshiW) [reply en/docs/tutorials/create-your-first-rest-api.md]: Done.

File: en/docs/tutorials/create-your-first-rest-api.md [hunk 12]
```diff
@@ -52,78 +56,81 @@ The `Statistics` REST API opens on a separate page.
...
+    By adding `string country` within square brackets in the path, you are introducing `country` as a path parameter for which the value should be in the string format.
 
-        By adding `string country` within square brackets in the path, you are introducing `country` as a path parameter for which the value should be in the string format.
+    The specified return type allows the REST API to return the output in either JSON or error? format.
 
-    4. Click **Advanced** and in the **Return Type** field, enter `json|error` to get the return in `json` or `error1` format.
+    Click **Save**.
 
-    5. Click **Save API**.
+    The low-code diagram of this resource opens .
```
[the remark this answers is not in the document]
Comment (RukshiW) [reply en/docs/tutorials/create-your-first-rest-api.md]: Done.

File: en/docs/tutorials/create-your-first-rest-api.md (hunk 11 above)
Comment (RukshiW) [reply en/docs/tutorials/create-your-first-rest-api.md]: Done.

File: en/docs/tutorials/create-your-first-rest-api.md (hunk 9 above)
Comment (RukshiW) [reply en/docs/tutorials/create-your-first-rest-api.md]: Done.

File: en/docs/tutorials/create-your-first-rest-api.md (hunk 8 above)
Comment (RukshiW) [reply en/docs/tutorials/create-your-first-rest-api.md]: Checked and found that it is used in two places, and corrected it in both the places.
